### PR TITLE
added code to automatically pull new warning types 

### PR DESCRIPTION
**Pull request by kdobolyi**
Uses Mona's code in live_data.py to create a local cache (.pickle) of all the warning types dynamically.

The HTML files will still need to be updated manually to handle new types,  but someone with even a slight but more js knowledge than me can probably do this easily by refactoring single_package.html and top_warnings.html

### CODE REVIEW

File: aura-borealis-flask-app/app.py [hunk 1]
```diff
@@ -18,13 +18,16 @@
 from live_data import get_all_warnings_counts_x
```
Comment (jspeed-meyers) [inline aura-borealis-flask-app/app.py]: @kdobolyi, when importing multiple functions from the same module in Python, I find [this syntax](https://github.com/IQTLabs/GitGeo/blob/d1ce3361dd279a29bf75687b4bb54145eaa93caa/test_gitgeo.py#L15) more concise. Consider refactoring.

File: aura-borealis-flask-app/app.py [hunk 2]
```diff
@@ -34,61 +37,11 @@ def get_user_selected_warnings(request):
 		collects all the checked checkboxes from a POST request used on various forms, 
 		and returns them as a list of strings
 	'''
+
 	warning_types_selected = []
-	if request.form.get('LeakingSecret') != None:
```
Comment (jspeed-meyers) [inline aura-borealis-flask-app/app.py:38]: Nice refactoring! You made this section so much more concise and just as understandable.

File: aura-borealis-flask-app/app.py [hunk 3]
```diff
@@ -520,13 +473,21 @@ def loadData(name):
 		cacheData(init_all_unique_warnings, 'init_all_unique_warnings')
 		cacheData(init_all_severities, 'init_all_severities')
 		cacheData(init_all_percentiles, 'init_all_percentiles')
-		print('init_all_percentiles', init_all_percentiles)'''
+		print('init_all_percentiles', init_all_percentiles)
+
+		all_warning_types = get_unique_warning_types()
```
Comment (jspeed-meyers) [inline aura-borealis-flask-app/app.py:481]: @kdobolyi, could you please add a brief comment at the top of this code chunk explaining what is going on?

**Review by jspeed-meyers — APPROVED** [review]

File: aura-borealis-flask-app/app.py (hunk 1 above)
Comment (kdobolyi) [reply aura-borealis-flask-app/app.py]: done!

File: aura-borealis-flask-app/app.py (hunk 3 above)
Comment (kdobolyi) [reply aura-borealis-flask-app/app.py:481]: done!

**Pull request merged**